cpp/examples/all-sky/mo_garand_atmos_io.cpp

```diff
@@ -94,7 +89,105 @@ void read_atmos(std::string input_file, real2d &p_lay, real2d &t_lay, real2d &p_
   io.close();
 }
 
+#ifdef RRTMGP_ENABLE_KOKKOS
+void read_atmos_kokkos(std::string input_file, real2dk &p_lay, real2dk &t_lay, real2dk &p_lev, real2dk &t_lev,
+                       GasConcs &gas_concs, real2dk &col_dry, int ncol) {
+  using Kokkos::parallel_for;
+
+  conv::SimpleNetCDF io;
+  io.open(input_file , conv::NETCDF_MODE_READ);
+
+  int nlay = io.getDimSize("lay");
+  int nlev = io.getDimSize("lev");
+
+  p_lay = real2dk("p_lay",ncol,nlay);
+  t_lay = real2dk("t_lay",ncol,nlay);
+  p_lev = real2dk("p_lev",ncol,nlev);
+  t_lev = real2dk("t_lev",ncol,nlev);
+
+  real2dk tmp2d;
+  // p_lay
+  io.read(tmp2d,"p_lay");
+  parallel_for( ncol, KOKKOS_LAMBDA (int icol) {
```

Comment:
I wonder if certain compilers (perhaps in particular situations) may not understand which ||for you want: yakl's or kokkos's. It is possible the user may issue something like `using yakl::c::parallel_for;` somewhere in a previous header.

Perhaps it's best to keep things explicit, and use `Kokkos::parallel_for`.It's only few extra chars.

Reply:
Agree.